org-roam-buffer.el

```diff
@@ -30,6 +30,7 @@
 ;; This library provides the org-roam-buffer functionality for org-roam
 ;;; Code:
 ;;;; Library Requires
+(require 'subr-x)
```

Comment:
```suggestion
(eval-when-compile (require 'subr-x))
```
From the commentary of subr-x.el:
> ;; NB If you want to use this library, it's almost always correct to use:
;; (eval-when-compile (require 'subr-x))

Reply:
+1. @juergenhoetzel I'll merge this in once you make this small change